src/main/java/frc/modes/DriverControl.java

```diff
@@ -36,9 +40,13 @@ private void driveSwerve() {
         double y = JoystickProfile.applyDeadband(-Input.SWERVE_XBOX.getX(Hand.kLeft));
         // double y =
         // JoystickProfile.applyDeadband(Input.SWERVE_XBOX.getX(Hand.kRight));
+
+        // magnify the x and y values?
```

Comment:
I agree that this code is confusing. Let's flag this for a follow-up session, @acrantel and @audrey-xie